src/main/resources/theme/rating_changed_label.fxml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.scene.control.Label?>
+<Label xmlns:fx="http://javafx.com/fxml"
+       xmlns="http://javafx.com/javafx"
+       fx:controller="com.faforever.client.game.RatingChangedLabelController"
+       visible="false"
+       fx:id="ratingChangedLabelRoot"
+       styleClass="rating-changed-label">
+</Label>
```

Comment:
I usually don't set `visible="false"` in FXML because it makes the element hidden when you edit it in SceneBuilder. Instead, I keep all elements visible and set them invisible in their controller's initialize method.